adam/language_specific/english/english_language_generator.py

```diff
@@ -193,7 +194,38 @@ def _translate_object_to_noun(
             if (dependency_node.part_of_speech != PROPER_NOUN) and (
                 MASS_NOUN not in lexicon_entry.properties
             ):
-                if count == 1:
+
+                # Objects that might possess the noun
+                try:
```

Comment:
I know it is supposedly "Pythonic", but I am not a fan of using exceptions for flow control.

I would suggest rewriting this section in the following way:
* first check if anything has a `HAS` relation with the object in question.
* if it does, then examine that particular object to see how to render it as a word.